datumaro/cli/contexts/project/diff.py

```diff
@@ -243,11 +245,11 @@ def save_item_bbox_diff(self, item_a, item_b, diff):
         _, mispred, a_unmatched, b_unmatched = diff
 
         if 0 < len(a_unmatched) + len(b_unmatched) + len(mispred):
-            if not item_a.has_image or not item_a.image.has_data:
+            if not isinstance(item_a.media, Image) or not item_a.media.has_data:
```

Comment:
Probably, it would be reasonable to add the same check for media type as in converters.

Reply:
Should we abort the save with an error if we have something other than an image? I think the check that we have now is enough.

Reply:
Currently, this class only works for images, which was essential previously. I think, we can just avoid calling image-specific parts, or disallow class use entirely and support it later. It is a good candidate into the list of necessary changes for this feature.

Reply:
Added a check before this function